src/test/java/org/trellisldp/event/EventSerializerTest.java

```diff
@@ -133,5 +139,7 @@ public void testSerializationStructureNoEmptyElements() throws Exception {
         assertTrue(AS.URI.contains((String) map.get("@context")));
 
         assertTrue(map.get("id").equals("info:event/12345"));
+        final String instant = map.get("published").toString();
+        assertTrue(map.get("published").equals(instant));
```

Comment:
See the comment above related to making this value-comparison test a little more robust.